gradle.properties

```diff
@@ -0,0 +1 @@
+android.sdk.channel=2
```

Comment:
I sure can, but still the `gradle.properties` file should not be ignored in Git, IMO.